### ISSUE TITLE
opacity is an unrecognized CSS property at line 0. Ignoring declaration.

### ISSUE THREAD

**Issue by mapleLeat** [danfickle/openhtmltopdf#226]
hello
Is the css property ''opacity" not supported?
Can you consider adding this support?

**Comment by danfickle** [comment]
No, not supported currently. It is however in neoFlyingSaucer (my previous failed attempt) so shouldn't be too hard to port over at some point. I've got to finish #180 first though.

**Comment by mapleLeat** [comment]
@danfickle 
Excuse me, opacity can support it now?

**Comment by gullerya** [comment]
Hi,

Any updated on this one?
As of now it looks like the whole range of transparency tools is not available, nor `opacity` neither `rgba` (just tried both of them with 2.0.16).

BTW: great library, cudos!

**Comment by conlaoch1** [comment]
Hi! Im also in need of working opacity. Could you please implement this feature? Thanks!

**Comment by allofapiece** [comment]
is it  not supported yet?

**Comment by zacharysiegel** [comment]
See [the discussion on project status](https://github.com/danfickle/openhtmltopdf/issues/921)

I created [this issue](https://github.com/openhtmltopdf/openhtmltopdf/issues/50) to track this in the new repository.
